src/NumericalAlgorithms/SphericalHarmonics/WignerThreeJ.hpp

```diff
@@ -0,0 +1,59 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <vector>
+
+/*!
+ * \brief Computes Wigner 3J symbols.
+ *
+ * Given l2, m2, l3, and m3, computes and caches the Wigner 3J symbols
+ *
+ * ( l1     l2 l3 )
+ * ( -m3-m2 m2 m3 )
```

Comment:
I doubt this renders the way you want. Can you check it? You might have to use latex mode